manifests/Folding@home/Folding@home/7.6.21.yaml

```
@@ -0,0 +1,18 @@
+Id: Folding@home.Folding@home
+Publisher: Folding@home
+Name: Folding@home
+AppMoniker: foldingathome
+Version: 7.6.21
+License: foldingathome.org Software License 
+Description: The Folding@home software allows you to share your unused computer power with us – so that we can research even more potential cures.
+Homepage: string # Valid secure URL for the package.
+Tags: folding protein
+Commands: list # List of commands or aliases the user would use to run the package.
+InstallerType: Inno
+Switches: # These can be used to change the install behavior if supported by the InstallerType.
+  Custom: /NORESTART
+  Silent: /SILENT

```

```suggestion
  Silent: /S
  SilentWithProgress: /S
```

I can't find any `SilentWithProgress`argument for the installer so I use `Silent` argument for it.